lib/fluent/plugin/filter_process_ucs_syslog.rb

```diff
@@ -48,8 +48,11 @@ def filter(tag, time, record)
       record["device"] = ""
       record["error"] = ""
 
-      # Filter out usernames
-      record["message"] = record["message"].gsub(/\\[-\w.]+/, "")
+      fullUsername = domain + "\\" + username
```

Comment:
the best practice is to use String interpolation, http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html

Reply:
good catch, done